roboteam_ai/src/control/shotControllers/ShotController.cpp

```diff
@@ -126,8 +126,30 @@ RobotCommand ShotController::goToPlaceBehindBall(world::Robot robot, const Vecto
 
 /// At this point we should be behind the ball. now we can move towards the ball to kick it.
 RobotCommand ShotController::moveStraightToBall(world::Robot robot, const std::pair<Vector2, Vector2>& lineToDriveOver) {
-    auto robotCommand = robot.getBasicPosControl()->getRobotCommand(std::make_shared<world::Robot>(robot),
-            lineToDriveOver.second);
+    /*
+     * Moving straight to the ball should be possible by driving forward at a small velocity as soon as you are at the
```

Comment:
you did your edits in 'movestraightToBall' which is not used when there is a ballsensor. Although this functionality does belong here it also needs to be in 'shoot'. Probably the functionality of 'shoot' and 'movestraighttoball' should be combined.